datasetinsights/evaluation_metrics/metrics_utils.py

```diff
@@ -0,0 +1,47 @@
+import collections
+
+import numpy as np
+
+
+def mean_metrics_over_iou(metric_per_iou):
+    """Calculate mean value over IOUs.
+
+    Args:
+        metric_per_iou (dict): metric records for each iou
```

Comment:
This method is more general than `average_over_iou`. It can apply to any metrics that return a dict of values such as `{label: float}`. Maybe we should have a general method that average over dict of values?